dev-lang/luajit/luajit-2.0.4.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
Please don't update dates on ebuilds you didn't touch. Well, unless somebody has touched it earlier and forgot to update the date…

Reply:
@mgorny fair.